tool/travis.sh

```diff
@@ -146,8 +141,8 @@ bin="${code}/bin"
 TARGET="${1}"
 COMPILER="${2}"
 COMPILER_VERSION="${3}"
-COMPILER_CONFIGURATION="${4}" # Debug or Release
-NIGHTLY="${5}" # Nightly is either true, false or conan
+BUILD_TYPE="${4}" # Debug or Release
+NIGHTLY="${5}" # Nightly is either true or false
```

Comment:
maybe 
```suggestion
IS_NIGHTLY_BUILD="${5}"
```
but I am not sure from the previous comment on what premises this is set to true and when it is set to false. Could you add a short doc comment on that?